lib/accountinformation/getUserFavoriteGames.js

```diff
@@ -0,0 +1,43 @@
+// Includes
+const http = require('../util/http.js').func
+
+// Args
+exports.required = ['userId']
+exports.optional = ['accessFilter', 'limit', 'cursor', 'sortOrder']
+
+// Docs
+/**
+ * ✅ Get the favorite games data.
+ * @category AccountInformation
+ * @alias getUserFavoriteGames
+ * @param {number} userId - The id of the user.
+ * @param {number} limit - The number of results per request.
+ * @returns {Promise<UserFavoriteGames>}
+ * @example
+ * const noblox = require("noblox.js")
+ * const userSocialLinks = await noblox.getUserFavoriteGames(172694510);
+**/
+
+function getUserFavoriteGames (userId, limit) {
```

Comment:
missing multiple parameters specified by the endpoint

see: https://games.roblox.com/docs/index.html